python/common/org/python/types/Set.java

```diff
@@ -417,28 +417,52 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
 
     @org.python.Method(
             __doc__ = "Return the difference of two or more sets as a new set.\n\n(i.e. all elements that are in this set but not the others.)",
-            args = {"other"}
+            args = {"other"},
```

Comment:
Hi, I don't think args is needed here, because according to docs this method only takes varags [(here)](https://docs.python.org/3.5/library/stdtypes.html#set.union) and single argument can be handled by tuple with one element.